README.md

```diff
@@ -155,11 +155,11 @@ inventory creation/modification:
 >>> import sphobjinv as soi
 >>> inv = soi.Inventory('doc/build/html/objects.inv')
 >>> print(inv)
-<Inventory (fname_zlib): sphobjinv v2.3, 220 objects>
+<Inventory (fname_zlib): sphobjinv v2.4, ... objects>
 >>> inv.project
 'sphobjinv'
 >>> inv.version
-'2.3'
+'2.4'
```

Comment:
Please revert to 2.3. I'll update this when the time comes, as part of that release process.